models/pathology_tumor_detection/configs/metadata.json

```diff
@@ -1,7 +1,8 @@
 {
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
-    "version": "0.5.1",
+    "version": "0.5.2",
     "changelog": {
+        "0.5.2": "update TensorRT descriptions",
```

Comment:
Should the monai version of this bundle be changed to `1.2.0rc5`? Same question for other bundles that don't change to rc5.
Thanks,
Bin